iamy/awsdiff.go

```diff
@@ -282,6 +283,13 @@ func (a *awsSyncCmdGenerator) updateRoles() {
 					"--policy-arn", a.to.Account.policyArnFromString(p))
 			}
 
+			// update max session duration
+			if fromRole.MaxSessionDuration != toRole.MaxSessionDuration {
+				a.cmds.Add("aws", "iam", "update-role",
+					"--role-name", toRole.Name,
+					"--max-session-duration", strconv.Itoa(toRole.MaxSessionDuration))
```

Comment:
I've always checked for err when using Itoa.   I guess we are safe enough here given the source of the data, but if you are feeling enthusiastic maybe add the check?

Reply:
Itoa [doesn't return an error](https://pkg.go.dev/strconv#Itoa)

Reply:
My bad,  mixed it up with atoi which does.